api/alembic/versions/2cc32257e8bc_rethink_classification.py

```diff
@@ -0,0 +1,71 @@
+"""rethink classification
+
+Revision ID: 2cc32257e8bc
+Revises: ef2482f08074
+Create Date: 2022-09-07 16:32:50.568143
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.orm.session import Session
+import geoalchemy2
+
+# revision identifiers, used by Alembic.
+revision = '2cc32257e8bc'
+down_revision = 'ef2482f08074'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic ###
+    op.create_table('advisory_classified_hfi',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('hfi', sa.String(), nullable=False),
+                    sa.Column('run_type', sa.Enum('forecast', 'actual', name='runtypeenum'), nullable=False),
+                    sa.Column('run_date', sa.Date(), nullable=False),
+                    sa.Column('for_date', sa.Date(), nullable=False),
+                    sa.Column('geom', geoalchemy2.types.Geometry(geometry_type='POLYGON', srid=3005,
+                                                                 spatial_index=False, from_text='ST_GeomFromEWKT', name='geometry'), nullable=True),
+                    sa.PrimaryKeyConstraint('id'),
+                    comment='HFI classification for some forecast/advisory run on some day, for some date'
+                    )
+    op.create_index('idx_advisory_classified_hfi_geom', 'advisory_classified_hfi',
+                    ['geom'], unique=False, postgresql_using='gist')
+    op.create_index(op.f('ix_advisory_classified_hfi_id'), 'advisory_classified_hfi', ['id'], unique=False)
+    op.create_index(op.f('ix_advisory_classified_hfi_run_type'), 'advisory_classified_hfi', ['run_type'], unique=False)
+    op.drop_index('ix_advisory_fire_zones_id', table_name='advisory_fire_zones')
+    op.drop_index('ix_advisory_fire_zones_mof_fire_zone_id', table_name='advisory_fire_zones')
+    op.drop_table('advisory_fire_zones')
+    op.drop_index('idx_advisory_shapes_geom', table_name='advisory_shapes')
+    op.create_index('idx_advisory_areas_geom', 'advisory_shapes', ['geom'], unique=False, postgresql_using='gist')
+    # ### end Alembic commands ###
+    # Advisory shapes was missing an SRID:
+    session = Session(bind=op.get_bind())
+    session.execute("SELECT UpdateGeometrySRID('advisory_shapes','geom',3005);")
+
+
+def downgrade():
+    # ### commands (mostly) auto generated by Alembic ###
+    op.drop_index('idx_advisory_areas_geom', table_name='advisory_shapes', postgresql_using='gist')
+    op.create_index('idx_advisory_shapes_geom', 'advisory_shapes', ['geom'], unique=False, postgresql_using='gist')
```

Comment:
I'm confused why you're creating this index as part of the downgrade() when the exact same index is created as part of the upgrade()?

Reply:
Good catch - the Index in the model file is wrong! I'll try fix it.